src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.ts

```diff
@@ -109,6 +125,10 @@ export class CustomerListComponent implements OnInit, OnDestroy, AfterViewInit {
   }
 
   deleteCustomer() {
+    if (this.idToDelete === this.currentCustomerIdToEdit) {
+      this.store.dispatch(new ResetCustomerToEdit());
+      this.resetProjectFieldsToEdit();
+    }
```

Comment:
I think that is code sequence in line 114 is exactly to this by the exception `this.resetProjectFieldsToEdit();` & `this.idToEdit`
```typescript
if (this.currentCustomerIdToEdit === this.idToEdit) {
  this.store.dispatch(new ResetCustomerToEdit());
}
```
Is there any way to extract them to a method?